README.md

```diff
@@ -62,25 +62,26 @@ The `Dockerfile` contained in the root of this respository has an example ROS2 F
 
 ```
 docker build -t ds4_driver/foxy .
+```
+
+After building the image you can automatically run the container by firstly adding the following line to your `~/.bashrc`
+file:
+
+```
+export DS4_DRIVER_LOCAL_PATH=<the path to your cloned version of ds4_driver>
```

Comment:
Since the `ds4_driver` repo is cloned when building the docker image, I don't think this env variable is required.

Reply:
If you want to be able to specify a local cloned version of `ds4_driver` instead of the one inside the docker image, maybe make it an optional command line argument with `$1`?

Reply:
Correct me if I am wrong but by looking at the provided `Dockerfile` I see that actually when building the container we clone the `ds4drv` repo not the `ds4_driver`. The `ds4_driver` repo is actually copied from where the `Dockerfile` resides, which I believe is ran from inside the package. How I have been able to make this work is that I took a local clone of the `ds4_driver` and then ran the Dockerfile from there. I don't think the actual `ds4_driver` is cloned from the Dockerfile. Why I wanted that env variable is to be able to mount the whole package source inside the container (from the locally cloned version which to my assumptions is the only version that we have to work with) and then more easily develop on it with the code changes being automatically reflected inside the running container. This is helpful only for development purposes. For people that just want to use the package I believe it is not needed. I can remove that env var thing from the bringup if you think it would be better. :)

Reply:
Oops, my mistake! You're right: ds4_driver isn't cloned when building the docker image. I might modify the Dockerfile some time but for the time being, mounting the local clone of `ds4_driver` is good. Although I would prefer using `$1` over environment variables, so if you could make that change that would be awesome!

Reply:
Resolved this according to suggestions!